src/content/content_recommender.py

```diff
@@ -1,18 +1,34 @@
+import numpy as np
+from sklearn.feature_extraction.text import TfidfVectorizer
+from sklearn.metrics.pairwise import cosine_similarity
 class ContentRecommender:
 
     def __init__(self):
-        pass
+        self.word_tfidf = TfidfVectorizer(ngram_range=(1, 1), analyzer='word', sublinear_tf=True, max_df=0.5, stop_words='english')
 
 #TODO - Fit - form a vocab using training events(all valid events).
 #TODO - Ideas for vocab (1 gram, 2 gram or ngram).
     def train(self, training_events):
         ## input : dict([user, list of events])
         ## output : dict([user, tfidf - vector])
+        # Assumed training events to be a list of event_info objects
+        training_events = np.array([event.description for event in training_events])
```

Comment:
Q. I mean how do you think we will supply data to this function.?
I was thinking to give dictionary as a input which will have user to list of event ids as the list. The partition would be done in index.py

Q. And what are those classes in data-model for ?
We made those classes first but later we decided we will not use unless there is a good reason to use them. One of the main reason to remove those classes is it is making the implementation complex and make debugging tough. If there is a good reason for using classes which cannot be done by dictionaries, we should surely use classes. I am very much in favor of classes because that's the standard of writing the code.